### DISCUSSION TITLE
Bundle model lung detection + OHIF Viewer

### DISCUSSION THREAD

**Discussion by junxiant** [Project-MONAI/MONAILabel#1431] [Q&A] [answered]
Hello,

I'm assuming the detection model from the bundle does not work on OHIF Viewer 3.5.1? 

Running the monailabel server for the detection model works just fine,
![image](https://github.com/Project-MONAI/MONAILabel/assets/25401067/77109ed2-025e-438c-be92-b4076c5abaf4)

But since it's a detection model,
![image](https://github.com/Project-MONAI/MONAILabel/assets/25401067/116edff4-8f45-4403-b2c3-2d19d69119d7)

When i opened up the OHIF-Viewer, detection doesnt seem to show up here. 
![image](https://github.com/Project-MONAI/MONAILabel/assets/25401067/61ab51e8-6a1c-4785-8198-01fa631a6ea1)

I'm guessing this is a front-end stuff as the plugin did not include detection under the drop down list?

**Comment by diazandr3s** [comment]
Hi @junxiant,

That's correct. It is a plugin/viewer limitation.
At the moment, the detection model works in Slicer only: https://github.com/Project-MONAI/MONAILabel/issues/1087

Tagging @tangy5 for visibility  :)

**Comment by tangy5** [comment, marked as answer]
HI @junxiant , thanks for your interest of the detection integration with MONAI Label. Currently, we only support 3D Slicer for the lung detection task. Please refer to this tutorial for a demo:
https://github.com/Project-MONAI/tutorials/blob/main/monailabel/monailabel_monaibundle_3dslicer_lung_nodule_detection.ipynb

For OHIF, we currently do not have the implementation yet. Feel free to give suggestions or implementation, it's a TODO item, and we hope more viewers can support a variety of medical applications.

**Comment by junxiant** [comment]
Thanks, awesome with the OHIF V3 integration, i think 3D Slicer works well for now as it can do both detection + segmentation labeling. The idea for OHIF viewer to have both these functions is so that radiologists can do these annotations online/remotely, which 3D slicer can't (Last i checked the best way would be to run it in a VM/AWS appstream). I'll keep a lookout when detection gets integrated into ohif some day
